Source/Scene/GltfFeatureMetadataLoader.js

```diff
@@ -131,7 +131,7 @@ GltfFeatureMetadataLoader.prototype.load = function () {
       var textures = results[1];
       var schema = results[2];
 
-      that._featureMetadata = new FeatureMetadata({
+      that._featureMetadata = parseFeatureMetadata({
```

Comment:
While diagramming the current architecture, it looks like I'll need to update `Gltf3DTileContent` as well.